### DISCUSSION TITLE
/\/*\web.securenetsystems.net/

### DISCUSSION THREAD

**Discussion by dportvine** [uBlockOrigin/uAssets#25816] [Q&A] [answered]
`https://streamdb7web.securenetsystems.net/ce/947`

Google Chrome 130.0.6723.70, uBO 1.60.0

<details>
<summary>Screen</summary>

![2024-10-27_164426](https://github.com/user-attachments/assets/d6bfb157-c193-447e-877a-8e92738f7e97)
![2024-10-27_164458](https://github.com/user-attachments/assets/e95cfd8b-6e57-42a0-8930-a54a5e7545fd)
</details>

Firefox 132, uBO 1.60.1rc0

<details>
<summary>Screen</summary>

![2024-10-27_164544](https://github.com/user-attachments/assets/0c4314c5-fc63-41fb-9cfc-f346b753628d)
![2024-10-27_164618](https://github.com/user-attachments/assets/a3e1dce5-b462-4148-b433-15552eb8f882)
</details>

Why I see an error in Firefox?

`||connect.facebook.net^$script,from=/\/*\web.securenetsystems.net/`

**Comment by gorhill** [comment]
Can you paste the filter which is erroring here? Transcribing from the screenshot is kind of tedious.

**Reply by ghost** [reply]
||connect.facebook.net^$script,from=/\/*\web.securenetsystems.net/

**Reply by gorhill** [reply, marked as answer]
The parser for regex-based domain doesn't expect `/` characters inside the regex, as these never appear in a hostname. From your screenshot, it appears what you want is really (edited to remove pointless leading `.*`):

    ||connect.facebook.net^$script,from=/web\.securenetsystems\.net$/

I do see the same error in Chromium:

![image](https://github.com/user-attachments/assets/850c7d3e-958f-4d97-9b7d-940cb88e042e)

---

I see no error in 1.60.0 because the parser for network filter option was re-worked to better detect option boundaries. It 1.60.1+ it ends up extracting `/\/` as the regex, which is an invalid regex.

**Reply by dportvine** [reply]
>```||connect.facebook.net^$script,from=/.*web\.securenetsystems\.net$/```

Tnx, this works for me.

**Reply by gorhill** [reply]
By the way, why not just use:

    ||connect.facebook.net^$script,from=securenetsystems.net

Avoiding regex whenever possible is preferable if this accomplish desired result.

**Reply by dportvine** [reply]
Yes, `securenetsystems.net` does not have a request to `facebook.net`

**Reply by stephenhawk8054** [reply]
`||connect.facebook.net^$script,from=securenetsystems.net` will be applied to all subdomains of `securenetsystems.net`.
